api/responses/get-case-risk.json

```diff
@@ -6,13 +6,21 @@
     "gender": "Male",
     "crn": "A12345"
   },
-  "riskOfSeriousHarm": {
-    "overallRisk": "VERY_HIGH",
-    "riskToChildren": "LOW",
-    "riskToPublic": "VERY_HIGH",
-    "riskToKnownAdult": "MEDIUM",
-    "riskToStaff": "HIGH",
-    "lastUpdated": "2021-10-09"
+  "roshSummary": {
+    "lastUpdatedDate": "2021-10-09T01:02:03.123Z",
+    "whoIsAtRisk": "Public:\nMr Edwin poses a very high risk to members of the public, especially if he believes they have mocked him, he has been drinking/taking drugs and is surrounded by his associates in the Westside Gang.\n\nKnown adult:\nMr Edwin poses a medium risk to his victim Wallce Stanfield. He has shown no desire for retribution but is unpredictable and resolves problems through violence if taking drugs and surrounded by peers.\n\nStaff:\nMr Edwin poses a high risk to probation staff as he resorts to violence if he feels undermined or not listed to.",
+    "natureOfRisk": "Mr Edwin has a pattern of resorting to violent behaviour if he feels humiliated/threatened/stressed. He has been known to use weapons or opportunistically looks for a way to cause the victim maximum harm. His risk increases greatly if he has been taking alcohol or drugs, which is likely to do if he feels like he is 'out of control' of his life.",
+    "riskImminence": "If Mr Edwin has had recent changes in his life that cause him to take drugs or drink; if he is surrounded by criminal peers/associates; if he is in a public place where he feels like he needs to protect his reputation.",
+    "riskIncreaseFactors": "Factors include:\n- relationship breakdown\n- unstable accommodation situation/lack of stable housing\n- association with criminal peers/associates\n- a lack of steady employment which causes him to return to drug dealing to make money\n- drug and alcohol use",
+    "riskMitigationFactors": "Factors include:\n- engagement in interventions related to mood/behaviour control\n- avoidance of drugs and alcohol\n- steady employment\n- stable accommodation\n- sources of support and value that are separate from being part of a gang/maintaining reputation e.g. family engagement, job, education",
+    "riskOfSeriousHarm": {
```

Comment:
This part of the contract looks wrong. See ticket https://dsdmoj.atlassian.net/browse/MRD-730 for the latest version.